src/test/compile-fail/rfc-2008-non-exhaustive/auxiliary/structs.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(non_exhaustive)]
```

Comment:
Unnecessary feature, `feature(non_exhaustive)` is needed only in crates defining `non_exhaustive` things.

Reply:
Whoops, wrong place for the comment.